docker-compose.yml

```diff
@@ -52,6 +52,7 @@ services:
       - ACTIVATE_POSTMAN_FALLBACK=
       - BULK_UPLOAD_MAX_NUM=1000
       - BULK_UPLOAD_RANDOM_STR_LENGTH=8
+      - API_KEY_SALT=$$2b$$10$$9rBKuE4Gb5ravnvP4xjoPu
```

Comment:
interesting read, since our repo is public (meaning anyone can read how we do auth), so I would rather be on a safe side of the security. 
Without the salt, it makes it easier for an attacker to reserve-eng the APIKey if they manage to penetrate the DB and get the hash.